### PR TITLE
Importing command from text

### PR DESCRIPTION
**Pull request by ZB63**
Sebastian Pawełoszek 215844
Patryk Stępień 215886

### CODE REVIEW

File: src/main/java/edu/kis/powp/jobs2d/command/gui/CommandImportWindow.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        c.fill = GridBagConstraints.BOTH;
+        c.weightx = 1;
+        c.gridx = 0;
+        c.weighty = 1;
+        content.add(btnLoadCommand, c);
+    }
+
+    private void loadCommand() {
+    	commandManager.setCurrentCommand(reader.read(currentCommandField.getText()));
+    	//CommandsFeature.getDriverCommandManager().setCurrentCommand(reader.read(currentCommandField.getText()));
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/gui/CommandImportWindow.java]: This comment should be removed

File: src/main/java/edu/kis/powp/jobs2d/features/Readers/SimpleFormatReader.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+public class SimpleFormatReader implements Reader {
+
+    @Override
+    public ComplexCommand read(String contentToParse) {
+
+        // W TAKI SPOSOB TRZEBA WPISYWAC KOMENDY!!!
+        // setPosition,1,1;
+        // operateTo,10,10;
+        // operateTo,100,30;
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/features/Readers/SimpleFormatReader.java]: Remove or change to method JavaDoc

File: src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java [hunk 3]
```diff
@@ -65,14 +68,23 @@ private static void setupDrivers(Application application) {
 
 		driver = new LineDriverAdapter(drawerController, LineFactory.getSpecialLine(), "special");
 		DriverFeature.addDriver("Special line Simulator", driver);
+		
+		application.addTest("Custom Command", action -> CommandsFeature.getDriverCommandManager()
+				.getCurrentCommand().execute(DriverFeature.getDriverManager().getCurrentDriver()));
+
```
Comment (kowallus) [inline src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java]: application.addTest("Run command", new SelectRunCurrentCommandOptionListener(DriverFeature.getDriverManager()));

File: src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java [hunk 4]
```diff
@@ -65,14 +68,23 @@ private static void setupDrivers(Application application) {
...
 		DriverFeature.updateDriverInfo();
 	}
 
 	private static void setupWindows(Application application) {
+		
+		Reader reader = new SimpleFormatReader();
```
Comment (kowallus) [inline src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java]: Please, move this line just before:
CommandImportWindow commandImportWindow = new CommandImportWindow(CommandsFeature.getDriverCommandManager(), reader);

**Review by kowallus — APPROVED** [review]
Approved, but please, fix issues pointed out before merging

File: src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java (hunk 3 above)
Comment (ZB63) [reply src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java]: I'm not sure what i was supposed to do.
These two lines do the same thing:

-application.addTest("Run command", new SelectRunCurrentCommandOptionListener(DriverFeature.getDriverManager()));

-application.addTest("Custom Command", action -> CommandsFeature.getDriverCommandManager()
.getCurrentCommand().execute(DriverFeature.getDriverManager().getCurrentDriver()));

but "Run Command" was already present in "User Test" tab, so i only removed our line.

**Pull request merged**